app/src/main/java/com/keylesspalace/tusky/fragment/ViewThreadFragment.java

```diff
@@ -612,32 +628,63 @@ private void handleStatusComposedEvent(StatusComposedEvent event) {
 
         if (eventStatus.getInReplyToId().equals(thisThreadsStatusId)) {
             insertStatus(eventStatus, statuses.size());
+            addReplyCount(0, 1);
```

Comment:
This looks like a bug? The detailed status is not guaranteed to be the first one in the list